packages/aws-cdk/lib/notices.ts

```diff
@@ -84,32 +84,105 @@ export interface NoticesFilterFilterOptions {
   readonly bootstrappedEnvironments: BootstrappedEnvironment[];
 }
 
-export class NoticesFilter {
+export abstract class NoticesFilter {
```

Comment:
To make it very clear that this class doesn't have any instance methods on purpose.